src/hotspot/cpu/x86/sharedRuntime_x86_32.cpp

```diff
@@ -1483,6 +1475,7 @@ nmethod* SharedRuntime::generate_native_wrapper(MacroAssembler* masm,
   // Frame is now completed as far as size and linkage.
   int frame_complete = ((intptr_t)__ pc()) - start;
 
+  // FIXME: The logic below do not apply anymore. Should we change anything? /ihse
```

Comment:
This file is now Linux only, so we should be able to remove any Windows special code. Someone with better knowledge about the product needs to confirm that the comment is indeed correct, and that this was only needed on Windows.

Reply:
Nah, leave it as is. Let's not regress native stubs unnecessarily, and this whole file would be gone after we deprecate 32-bit port completely.